lib/enigma.rb

```diff
@@ -37,4 +37,35 @@ def shifts(key, date)
     end
     shifts
   end
+
+  def to_indexes(string)
+    alphabet = Array("a".."z")<< " "
+    string_array = string.downcase.chars
+    string_array.map do |char|
+      alphabet.find_index(char)
+    end
+  end
```

Comment:
`to_indexes` takes a string to be changed, down cases it, then turns it into and array of the characters which make up the string.
Once we have that array we can then turn it into an array of numbers based on the index position of each character in our `alphabet` array. 
Side note `alphabet` will show up in more than one method, it should probably become an instance variable.